Jenkinsfile

```diff
@@ -14,6 +14,10 @@
 // I.e. for testing library changes
 @Library(value="pipeline-lib@fewer_tests") _
 
+boolean is_master_build() {
```

Comment:
Is this purpose of this function really (only) know if it's _master_, or if it's a release branch also?  i.e. `release/*`.